include/PI/p4info/tables.h

```diff
@@ -104,12 +115,19 @@ const pi_p4_id_t *pi_p4info_table_get_actions(const pi_p4info_t *p4info,
                                               pi_p4_id_t table_id,
                                               size_t *num_actions);
 
+//! Returns NULL if @action_id is not valid for the table.
+const pi_p4info_action_info_t *pi_p4info_table_get_action_info(
+    const pi_p4info_t *p4info, pi_p4_id_t table_id, pi_p4_id_t action_id);
+
 bool pi_p4info_table_has_const_default_action(const pi_p4info_t *p4info,
                                               pi_p4_id_t table_id);
 
+// has_mutable_action_params is deprecated
+// it will always be set to false if the table has a const default action, true
```

Comment:
Question: Did you intend to day "true" instead of "false" in this line?

If not, can you explain why?

Reply:
No worries. I actually got it wrong the first time I wrote the comment and realized it only when submitting the PR :)